sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -213,6 +214,8 @@ def recognize_entities(  # type: ignore
         docs = _validate_input(documents, "language", language)
         model_version = kwargs.pop("model_version", None)
         show_stats = kwargs.pop("show_stats", False)
+        if self._string_index_type:
```

Comment:
This is me learning Python....
if the user for whatever reason finds out we have this parameter, and decides to pass it by using `kwargs` the library wouldn't read it and will always use the default value for the library, right?

Reply:
same if it is V3.0 of the service which I think this line wouldn't execute as `self._string_index_type` will be set to `None`

Reply:
yeah, we never read `string_index_type` from kwargs from users. It's just if the api version is not v3.0, we add string_index_type == "UnicodeCodePoint" to the kwargs for the actual network call